test/cljam/test_common.clj

```diff
@@ -223,9 +232,14 @@
 (def test-bcf-invalid-file "test-resources/bcf/invalid.bcf")
 (def test-bcf-no-samples-file "test-resources/bcf/test-no-samples.bcf")
 (def test-bcf-complex-file "test-resources/bcf/test-v4_3-complex.bcf")
+(def test-bcf-various-bins-file "test-resources/bcf/various-bins.bcf")
+(def test-bcf-various-bins-csi-file "test-resources/bcf/various-bins.bcf.csi")
+(def test-large-bcf-file (cavia/resource mycavia "large.bcf"))
+(def test-large-vcf-csi-file (cavia/resource mycavia "large.bcf.csi"))
```

Comment:
Rename this to `test-large-bcf-csi-file` as `test-large-vcf-csi-file` is defined at https://github.com/chrovis/cljam/pull/187/files#diff-ec7848941b3c1273617402827ddb7e2dR220

```suggestion
(def test-large-bcf-csi-file (cavia/resource mycavia "large.bcf.csi"))
```